virttest/qemu_devices/qcontainer.py

```diff
@@ -1019,6 +1019,10 @@ def pflash_handler(firmware_name, cmd):
             """
             machine_cmd = cmd
             devs = []
+            images = params.objects('images')
+            if not images:
+                return devs, machine_cmd
+
             firmware_path = params.get(firmware_name + "_path")
             if firmware_path:
```

Comment:
```suggestion
            if firmware_path and params.objects('images'):
```

Reply:
Maybe this also works and follows the original design that to handle `pflash` if `firmware_pathand` is not `None`,  and modify the fewest codes.
BTW, the codes you modified are ok for me, it's up to you.
Thanks.

Reply:
Indeed, already updated.